roles/build_image_bundle/tasks/main.yml

```diff
@@ -0,0 +1,109 @@
+- name: Handle bundle image from source image
+  block:
+    - name: Set input image
+      set_fact:
+        bundle_image: "{{ operator_input_image | regex_replace('[A-z]*://', '') }}"
+        operator_name: "{{ operator_input_image.split(':')[0] | regex_replace('[A-z]*://', '') | basename }}"
+        operator_version: "{{ operator_input_image.split(':')[1] }}"
+    # - fail:
+    #     msg: "{{ operator_name }} {{ operator_version }}"
+    - name: Remove bundle image {{ bundle_image }}
+      shell: "{{ opm_container_tool }} rmi -f {{ bundle_image }}"
+  when: not operator_input_image_convert|bool
```

Comment:
Missing the default value for this, I think it needs to be set to 'false'.

Reply:
solved